src/Fleck/Interfaces/ISocket.cs

```diff
@@ -8,7 +8,7 @@
 
 namespace Fleck
 {
-    public interface ISocket

```

Comment:
I think it might be good idea to remove original `Dispose()` method from this interface as it exists in `IDisposable`.